labrad/backend.py

```diff
@@ -45,6 +47,13 @@ def disconnect(self):
             self._disconnect()
             self.connected = False
 
+    def spawn(self):
+        """Start a new independent backend connection to the same manager."""
+        inst = self.__class__(name=self.name)
```

Comment:
Just to be clear, `self.__class__` is the class object, not a method. Then we call this class object to create an instance, just like given a class `Foo` we call `Foo()` to create a new instance. I could split up this expression to make it more clear.